manala.maxscale/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Tests on minor versions
```

Comment:
Test related changes are not worth a changelog entry IMHO

Reply:
👍